schema/project-level/project-package-schema.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "http://standard.open-contracting.org/infrastructure/beta/project-package-schema.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Schema for an OC4IDS project package",
+  "description": "A package of OC4IDS projects. Note that all projects within a package must have a unique project identifier.",
+  "type": "object",
+  "required": [
+    "uri",
+    "publisher",
+    "publishedDate",
+    "releases",
```

Comment:
Replace "releases" with "projects".

Reply:
fixed in https://github.com/open-contracting/infrastructure/pull/89/commits/3e4ef5d291f10edeb207f19b38a0bbff0f4d5886